CHANGELOG.md

```diff
@@ -0,0 +1,245 @@
+# Changelog
+
+## [Unreleased](https://github.com/unit8co/darts/tree/develop)
+
+[Full Changelog](https://github.com/unit8co/darts/compare/0.2.3...develop)
+
+Changes suceptible to **break existing code** working on a previous release are marked with a "&#x1F534;".
```

Comment:
Can add one line of explanation that darts is still in early development and we still cannot guarantee backward-compatibility, sorry for the trobule etc. etc.

Reply:
Will add. Maybe we could also be a little bit more careful in the next releases as the library is getting more users, and mark as deprecated first when applicable ?

Reply:
👍